atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -226,12 +229,32 @@ def manifest_list_entries_match(self, image, build_id):
             v2_digest = archive['extra']['docker']['digests'][v2_type]
             koji_archives_data[arch] = v2_digest
 
-        if koji_archives_data == manifest_list_data:
-            self.log.info('Deeper manifest list check verified v2 manifest references match')
-            return True
-        self.log.warning('Manifest list refs "%s" do not match koji archive refs "%s"',
-                         manifest_list_data, koji_archives_data)
-        return False
+        platform_to_arch_dict = get_platform_to_goarch_mapping(self.workflow)
+        architectures = []
+        for arch in self.architectures:
+            architectures.append(platform_to_arch_dict[arch])
```

Comment:
I think renaming `self.architectures` to `self.platforms` might help avoid confusion

Reply:
Nice catch, fixed!